OpenRA.Game/Settings.cs

```diff
@@ -20,26 +20,66 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchArgs
```

Comment:
You don't seem to get what @pchote was enforcing here. He wanted me to auto-generate those parameters for a man-page so first they became settings so I could add `[Desc()]` tags, then I had to invent something new (non-persisting settings read from args) and now I am stuck in rebase and refactor hell. Will try to fix this shortly.